### ISSUE TITLE
infoPanel : new features

### ISSUE THREAD

**Issue by schmurtzm** [OnionUI/Onion#67]
Infopanel is the app designed to display text information and images. 

**These uses of Infopanel are already defined :**  

- various information screens (example : shutdown screen or the ports information panel when a file is missing)
- multiple pages manual (example : the user manual at the end of Onion's installation)
- screenshot viewer (to allow to see screenshots previously done with the shortcut "menu + power".)

--------

**To achieve these uses these command line args must be available :** 

- [x] "-t" or "--title" --> Title of the info screen
- [x] "-m" or "--message" --> display a message
- [x] "-i" or "--image" --> load one image
- [x] "-j" or "--images-json" --> load images from a json
- [x] "-a" or "--auto" --> display one image and automatically close after 2 seconds
- [x] "-d" or "--directory" --> load all images of a folder (ordered alphabetically) 
- [x] "-s" or "--show-theme-controls" --> show themed header/footer/navigation buttons/image number

Some examples : 
- for a file: `./bin/infoPanel -i "res/install_complete.png"`
- for a folder : `./bin/infoPanel -d "/mnt/SDCARD/Screenshots"`

--------

**Functionalities to add :** 

To be more comfortable we could add these features to Infopanel :
- [x] Be able to navigate backward in the manual ([solution ref](https://discord.com/channels/@me/996705786493407332/1007253567809257603), [ref2](https://discord.com/channels/@me/996705786493407332/1007255418227142676))
- [x] Be able to use LEFT/RIGHT to navigate
- [x] Menu button to exit directly

--------

**Optional additional features / ideas :** 
- [ ] A new arg -p ("pages") to display the current number of the picture (exemple 1/17)
- [ ] A new arg -r ("reverse") to invert the displayed order
- [ ] A new arg -f ("footer") to display the standard controls at bottom
- [ ] Accept pargs so we could just use this command to feed it an entire folder:
`find somefolder -maxdepth 1 -print0 |xargs -0 ./bin/infoPanel`
- [ ] Make it possible to accept multiple files from different folders :
`./bin/infoPanel res/page1.png wallpaper/sunpng screenshots/MainUI.png res/page4.png`

**Comment by introkun** [comment]
`Be able to navigate backward in the manual` will be resolved by #53

**Comment by introkun** [comment]
@schmurtzm requirements contain non-final statemens (about navigation, about name of the application).
Why do we need pargs and/or args for images? Current solution in #53 is based on json file with list of images to show. If we want to change this behaviour I need to know what is the final one.

**Comment by schmurtzm** [comment]
@introkun, I have updated the description, I'm not a specifications writer 😉  but it will be probably more clear like this.

**Comment by introkun** [comment]
>  "-j" or "--images-json" --> load images from a json
Lets add format of the json for this argument:
```
{
  "images": [
    { "path": "./page1.png" },
    { "path": "./page2.png" },
    { "path": "./page3.png" },
    { "path": "./page4.png" }
  ]
}
```

**Comment by introkun** [comment]
#105 resolves navigation by left/right and exit by the menu button

**Comment by introkun** [comment]
Image browsing in directory will be implemented in #106

**Comment by introkun** [comment]
"-s" or "--show-theme-controls" --> show themed header/footer/navigation buttons/image number
"-d" or "--directory" --> load all images of a folder (ordered alphabetically)
implemented by #106
